src/Cms/Helpers.php

```diff
@@ -38,6 +38,13 @@ class Helpers
 		// TODO: switch to true in v6
 		'plugin-extends-root' => false,
 
+		// The `Content\Translation` class keeps a set of methods from the old
+		// `ContentTranslation` class for compatibility that should no longer be used.
+		// Some of them can be replaced by using `Version` class methods instead
+		// (see method comments). `Content\Translation::contentFile` should be avoided
+		//  entirely and has no recommended replacement.
+		'translation-methods' => false,
```

Comment:
`Todo`: I think we should be throwing deprecation warnings by default for this one as we intend to remove the methods in v6:

```suggestion
		'translation-methods' => true,
```